.gitguardian.yml

```diff
@@ -1,3 +1,7 @@
-matches-ignore:
-  - name: MIXPANEL_TOKEN
-    match: acd87c5a50b56df91a795e999812a3a4
+secret:
+  ignored-paths:
+    - 'packages/server/modules/emails/tests'
+  ignored-matches:
+    - match: acd87c5a50b56df91a795e999812a3a4
```

Comment:
so verbose! can't believe you have to repeat the token twice and even refer to the full path to file...

anyway, I'm guessing that's not something we can do much about, but the issue that can be resolved is the fact that there's an absolute path to `bootstrapper.ts` from your own workstation, which isn't going to work for anyone else. are relative paths supported?

Reply:
@iainsproat is the 'name' key just a display name/title for the rule? cause if so, we can make it shorter and more descriptive at the same time by just setting it to "mixpanel token". the token being put in the 'name' key doesn't help (we can already see it from the 'match' key) and the file name doesn't give us anything either cause we can always just do a full text search in the project to find where the token is used. it doesn't tell us what kind of secret/token it is either.